### ISSUE TITLE
Integrating Apache/MySQL into WinPython (like XAMPP / WAMPP) ? 

### ISSUE THREAD

**Issue by richlysakowski** [winpython/winpython#578]
Would it be possible for the WinPython developers to integrate MySQL and Apache into WinPython, like XAMPP or WAMPP?  

This would be really cool because then new developers would have the built-ins to build desktop AND web applications all in one WinPython.    

Make it easy for Pure Python web newbies to stay in WinPython the way PHPers can stay in XAMPP/WAMPP for web development.  I am sick of newbie web developers getting heavy PHP imprinting.

**Comment by stonebig** [comment]
MySQL and Apache do require a true installation, isn't it ? In that case it's not a wise target for WinPython..

**Comment by gilan65** [comment]
Not at all. XAMPP portable installer is not a true installation. Just now I use XAMPP together to WinPython on a pendrive, but I've two distinct folders... It would nice to have just a WinPython "installation", with MySQL and Apache, but without PHP and/or Perl...
